### PR TITLE
Update readme

### LINKED ISSUE #27: Plugin support
As I mentioned in #26 I'm working on a POC of some tests for https://shields.io/ (see badges/shields#927).

I'd like to add plugins for:

- A custom assertion
- Setting up and cleaning up mock requests using `nock`
    - Probably that will look like this: `.intercept(nock => nock(...).get(...))`

I noticed custom plugins were mentioned in the readme. Any thoughts on how the inheritance would be implemented, or libraries with good patterns to follow? Mixins could be a way to do it.

The nock plugin would also need a `before()` function to register callbacks, as a counterpart to `after()`, which seems straightforward enough.

### PR DESCRIPTION
**Pull request by paulmelnikow**
- Document plugin support (Closes #27) and link to icedfrisby-nock
- List maintainers and update contributing info
- Fix indentation and remove semis in code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,17 +39,17 @@ Each set of unique sequences or API endpoint tests should be started with new `f
 
 ```javascript
 
-var frisby = require('icedfrisby');  // get IcedFrisby with `npm install icedfrisby`
-var Joi = require('joi'); // get Joi with `npm install joi`
+var frisby = require('icedfrisby')  // Get IcedFrisby with `npm install icedfrisby`.
```
Comment (MarkHerhold) [inline README.md]: maybe change to `const`?

Comment (paulmelnikow) [inline README.md]: Good idea!

File: README.md [hunk 2]
```diff
@@ -39,17 +39,17 @@ Each set of unique sequences or API endpoint tests should be started with new `f
 
 ```javascript
 
-var frisby = require('icedfrisby');  // get IcedFrisby with `npm install icedfrisby`
-var Joi = require('joi'); // get Joi with `npm install joi`
+const frisby = require('icedfrisby')  // Get IcedFrisby with `npm install icedfrisby`.
+const Joi = require('joi') // Joi is installed along with IcedFrisby.
```
Comment (MarkHerhold) [inline README.md]: But you can't directly `require('joi')` unless you've installed it separately.

Comment (paulmelnikow) [inline README.md]: Hmm, I haven't had that problem… I'd agree it's probably not a good idea to require it unless it's declared separately, though.

I'll revert this comment change. Let's discuss further how to handle this.

**Pull request merged**